models/resnet_official.py

```diff
@@ -41,23 +46,23 @@ def __init__(
             groups: int = 1,
             base_width: int = 16,
             dilation: int = 1,
-            #norm_layer: Optional[Callable[..., nn.Module]] = None
-            norm = None
+            # norm_layer: Optional[Callable[..., nn.Module]] = None
```

Comment:
All the changes to the files in `models/` are just stylistic.  E.g. having a space after the `#` in comments.  Feel free to ignore.  The traditional standard for formatting python code is called [PEP8](https://www.python.org/dev/peps/pep-0008/).